components/Banner/examples/supplemental/MDCBannerView.h

```diff
@@ -14,19 +14,33 @@
 
 #import <UIKit/UIKit.h>
 
-@class MDCButton;
+#import "MaterialButtons.h"
 
 __attribute__((objc_subclassing_restricted)) @interface MDCBannerView : UIView
 
-@property(nonatomic, copy, nonnull) NSString *text;
+/**
+ A view that displays the text on a @c MDCBannerView.
+ @discussion These properties of @c textLabel can be used to configure the text shown on @c MDCBannerView.
```

Comment:
```suggestion
 @discussion The properties of @c textLabel can be used to configure the text shown on @c MDCBannerView.
```

here, and throughout.

Reply:
Addressed in 5ed33d9c7280f7d2798dab0b2aad4651983c8238.